t/009-log.t

```diff
@@ -265,7 +265,69 @@ qr/\[error\] \S+: \S+ \[lua\] set_by_lua:2: truefalsenil,/
 
 
 
-=== TEST 14: print() in header filter
+=== TEST 14: test table with metamethod
+--- config
+    location /log {
+        set_by_lua $a '
```

Comment:
Can we just use `content_by_lua_block {}` here? I think it's distraction to use `set_by_lua '... return 32'` and `echo $a` in these test cases :)